src/sota_tools/ostree_http_repo.cc

```diff
@@ -40,23 +42,48 @@ OSTreeRef OSTreeHttpRepo::GetRef(const std::string &refname) const { return OSTr
 
 OSTreeObject::ptr OSTreeHttpRepo::GetObject(const uint8_t sha256[32]) const { return GetObject(OSTreeHash(sha256)); }
 
+bool failure = false;
```

Comment:
Is it used somewhere?

Reply:
My bad, another debugging/rebasing holdover. Fixed.